influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -153,12 +154,14 @@ impl WriteBufferImpl {
         let persisted_files = Arc::new(PersistedFiles::new_from_persisted_snapshots(
             persisted_snapshots,
         ));
+        let parquet_cache = Arc::new(ParquetCache::new(&persister.mem_pool));
```

Comment:
It's odd to me that the persister would have a mem pool that would be shared with the cache. I would expect the persister to only have things in memory for as long as it takes to persist.

The cache should have its own memory pool, which should be bounded in size based on the configuration that the user provides. We can default to some percentage of the available RAM, but it should be bounded and separate from persistence.

We may not want to cache everything that gets persisted. For example, they could be writing to a bunch of tables that they never query, so we wouldn't want to bother caching those files at all.

Reply:
Agreed. I opened an issue as part of the parquet cache epic here: https://github.com/influxdata/influxdb/issues/25369. I don't think this is a very difficult change but I may address it separately depending on how this PR shakes out.